### PR TITLE
PLAT-143789: Added screenshot and UI tests for VideoPlayer

### PR DESCRIPTION
**Pull request by paul-beldean-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added screenshot and UI tests for VideoPlayer

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-143789

### Comments

Enact-DCO-1.0-Signed-off-by: Paul Beldean paul.beldean@lgepartner.com

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
Screenshot tests look good 👍

File: tests/ui/specs/VideoPlayer/VideoPlayer-specs.js [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+	});
+
+	describe('props', function () {
+		beforeEach(function () {
+			Page.open('Props');
+		});
+
+		it('should hide controls after 3 seconds', function () {
+			Page.delay(3500);
+			expect(videoPlayerProps.mediaControlsFrame.getCSSProperty('opacity').value).to.equal(0);
```
Comment (daniel-stoian-lgp) [inline tests/ui/specs/VideoPlayer/VideoPlayer-specs.js:206]: here maybe you can show that the controls are first displayed, then they dissapear

File: tests/ui/specs/VideoPlayer/VideoPlayer-specs.js [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+		});
+
+		it('should have title', function () {
+			Page.delay(1000);
+			expect(videoPlayerProps.title.getAttribute('innerHTML')).to.equal('Sandstone VideoPlayer Sample Video');
+		});
+
+		it('should hide title after 1 second', function () {
+			Page.delay(2200);
+			expect(videoPlayerProps.titleFrame.getCSSProperty('opacity').value).to.equal(0);
```
Comment (daniel-stoian-lgp) [inline tests/ui/specs/VideoPlayer/VideoPlayer-specs.js:227]: same here

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Comment by adrian-cocoara-lgp** [comment]
Closed in favor of WRN-17764

**Pull request closed without merging**